macros/AddonSidebar.ejs

```diff
@@ -12,145 +12,320 @@ function currentPageIsUnder(root) {
   return "closed";
 }
 
+var text = mdn.localStringMap({
+  'en-US': {
```

Comment:
We are horribly inconsistent throughout this code in single versus double quotes, but we should at least try to be consistent within each macro.

I think single quotes is better, so would you mind fixing up lines 1-13 of this file to use single quotes?

I see that we also use double quotes for HTML attributes. Perhaps we should leave those ones, that seems to be what we use on MDN.

Reply:
Fixed, and I agree, it is better to have consistency within a file.